src/main/server/start.sh

```diff
@@ -45,8 +47,11 @@ DB_USERNAME=
 DB_PASSWORD=
 LICENSE_KEY=
 
+# Set the default value for JVM_MEMORY_OPTIONS
+DEFAULT_MIN_HEAP_SIZE="1g"
+JVM_MEMORY_OPTIONS="-Xms$DEFAULT_MIN_HEAP_SIZE"
```

Comment:
No need to define `JVM_MEMORY_OPTIONS` options here. Just define in the comment below

Reply:
done